packages/ember-template-compiler/lib/index.js

```diff
@@ -35,7 +34,6 @@ registerPlugin('ast', AssertNoEachIn);
 
 if (!_Ember.ENV._ENABLE_LEGACY_VIEW_SUPPORT) {
   registerPlugin('ast', AssertNoViewAndControllerPaths);
-  registerPlugin('ast', AssertNoViewHelper);
```

Comment:
This likely should stick around (moved outside the `if`) to provide feedback to folks that might try to use `{{view 'foo'}}` after `{{view` keyword was removed.